aspnetcore/fundamentals/configuration/options/samples/2.x/OptionsSample/SampleApp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
The 2.2 web app project templates also add the following element to this ItemGroup:

```xml
<AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
```

Reply:
Yes, indeed. The template opts-in ... NOT the SDK. I'm still getting used to the new bits.